behaviors/IndexModelFormOperations.php

```diff
@@ -127,6 +129,24 @@ public function onModelFormGetModelFields()
         ];
     }
 
+    public function onModelShowAddDatabaseColumnsPopup()
+    {
+        $columns = ModelModel::getModelColumnsAndTypes($this->getPluginCode(), Input::get('model_class'));
+        $config  = $this->makeConfig($this->getAddDatabaseColumnsDataTableConfig());
```

Comment:
Aligning `=` should only be done inside of arrays please

Reply:
:ok_hand: fixed!